main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1237,9 +1237,10 @@ public NestedScrollView getDispatchedView(final ViewGroup parentView) {
             buttonWatchlistRemove.setOnClickListener(new RemoveFromWatchlistClickListener());
             updateWatchlistBox();
 
-            // WhereYouGo and ChirpWolf
+            // WhereYouGo,ChirpWolf and Adventure Lab
```

Comment:
There is a space missing:
`// WhereYouGo, ChirpWolf and Adventure Lab`

Reply:
You guys are taking the review really serious. I am exactly the same type of guy ;) Thanks for pointing it out. Will fix with next push.

Reply:
commit done